### DISCUSSION TITLE
添加启动国际基岩版的功能

### DISCUSSION THREAD

**Discussion by killcerr** [Meloong-Git/PCL#4233] [讨论]
https://github.com/killcerr/PCL2/tree/launch-bedrock
https://github.com/Hex-Dragon/PCL2/pull/4246
仅仅是做一个和官方启动器类似的功能，而不是类似BedrockLauncher或者类似产品

**Comment by MoYuan-CN** [comment]
你希望直接启动么。。。
现在你可以通过自定义主页打开 `minecraft://` 这个 URL scheme 来拉起 MCBE

**Reply by wuliaodexiaoluo** [reply]
然后弹出提示：请前往发烧平台启动游戏（

**Comment by allMagicNB** [comment]
#81

**Comment by pnpn111** [comment]
> #81

龙猫在这个issue里的理由是启动不了，但我最近发现了这个项目： https://github.com/BedrockLauncher/BedrockLauncher
这似乎是基岩版的启动器，而且是C#写的
这似乎意味着从.net平台启动基岩版成为了可能

**Reply by killcerr** [reply]
我觉得专业的事情还是要专业的去弄，BedrockLauncher已经出现好几年了，如果将其功能加入PCL2工程量会过大。这也是为什么我只是准备做一个和官方启动器类似功能的原因。

**Comment by xxtg666** [comment]
没必要吧，做基岩版启动器做的也就是一个多版本管理功能，但是 PCL2 是 Java 版的启动器，添加一个没多少人会用的功能显得很多余

**Comment by killcerr** [comment]
仅仅是做一个和官方启动器类似的功能，而不是类似BedrockLauncher

**Comment by Deep-Dark-Forest** [comment]
> 仅仅是做一个和官方启动器类似的功能，而不是类似BedrockLauncher

那直接自定义主页就行啊……

**Reply by killcerr** [reply]
![image](https://github.com/user-attachments/assets/53eb2664-6abe-454e-8e09-9becf171ed50)

我会用这种办法

**Comment by killcerr** [comment]
自定义主页应该不能检测基岩版是否被安装了吧，如果可以的话，那我这个计划确实没什么意义

**Comment by xxtg666** [comment]
那为什么不直接从开始菜单里启动呢……

（毕竟没多少人会想用 PCL2 启动基岩版。你需要可以在自己的自定义主页里添加，既然是自用了那也没必要检测装没装

**Reply by killcerr** [reply]
https://github.com/Hex-Dragon/PCL2/pull/4246

**Reply by killcerr** [reply]
只是因为官启有这个功能而已（）

**Comment by xzfg-n15421** [comment]
https://learn.microsoft.com/zh-cn/windows/uwp/debug-test-perf/loose-file-registration
好像只有这一个思路来着，用户必须以管理员身份去设置开启开发人员模式，并且开发者需要做一个appx包索引，~~毕竟这个官方可没有给版本列表~~
建议专业的事由专业的人来做，没必要加这个功能
